frame/election-provider-multi-phase/src/lib.rs

```diff
@@ -1565,12 +1596,16 @@ impl<T: Config> Pallet<T> {
 			.ok_or(ElectionError::<T>::NothingQueued)
 			.or_else(|_| {
 				<T::Fallback as ElectionProvider>::elect()
-					.map(|supports| ReadySolution {
-						supports,
-						score: Default::default(),
-						compute: ElectionCompute::Fallback,
-					})
 					.map_err(|fe| ElectionError::Fallback(fe))
+					.and_then(|supports| {
+						Ok(ReadySolution {
+							supports: supports
+								.try_into()
+								.map_err(|_| FeasibilityError::BoundNotMet)?,
```

Comment:
Should add some documentation to `type Fallback` about this condition: `Fallback` is not bounded, but it better be, and it better have a `MaxWinners` that is compatible with this pallet.

Reply:
Fallback is now bounded by associated type `MaxWinners`